common/ideologies/00_ig_ideologies.txt

```diff
@@ -46,14 +46,6 @@
 		law_protectionism = neutral
 		law_free_trade = disapprove
 	}
-	lawgroup_diplomatic_stance = {
-		law_expansionism = strongly_approve
-		law_imperialism = approve
-		law_armed_neutrality = approve
-		law_anti_imperialist = neutral
-		law_pacifist = neutral
-		law_true_neutrality = neutral
-	}
```

Comment:
we're keeping these, and even expand them in my patch

Reply:
I wasn't aware we wanted to keep them, on account of the fact that we were changing it to be a decision. are you saying we will be making their success to be changed viable now too then? because that was a major concern which led to them becoming decisions.

Reply:
Their purpose has changed, now we I just want the system to automatically apply opinion modifiers. So you enact by decision, but you still incur opinion penalties/bonusses based on the interest groups.